README.md

```diff
@@ -5,13 +5,25 @@ node-ipfs
 
 # Description
 
-This repo (will) contains the entry point for the Node.js implementation of IPFS spec, similar to go-ipfs. It also presents the roadmap a modules of node-ipfs and their current state.
+This repo will contain the entry point for the Node.js implementation of IPFS spec, similar to [go-ipfs](https://github.com/ipfs/go-ipfs). Right now, it holds the roadmap for the development of modules for node-ipfs, as well as their current state.
 
+We are building node-ipfs because it will inform how go-ipfs works, separate concerns, and allow multiple different streams at once. Most of the work for IPFS does happen elsewhere, but this is an equally important part of our roadmap to lead to a permanent, IPFSed web.
```

Comment:
- maybe remove the `and allow multiple different streams at once`.
- instead add `and allow a complete in-browser-tab implementation with no install friction`.